website/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/karpenter-provider-aws/website
 
-go 1.22
```

Comment:
So I stopped running into this and don't think we should pin if possible. Have you reinstalled go? If we do pin we should increase the pin to 1.22.1, IIRC 1.22.0 has some CVEs related to MIME parsing.